docs/user_guide/quickstart.rst

```diff
@@ -0,0 +1,168 @@
+Quickstart
+==========
+
+Generate a container
+--------------------
+
+Use the command :code:`neurodocker generate` to generate containers. In the examples below,
+we generate containers with `Nipype <https://nipype.readthedocs.io/en/latest/>`_,
+`Jupyter Notebook <https://jupyter.org/>`_, and `ANTs <https://github.com/ANTsX/ANTs>`_.
+
+Please see the :doc:`examples` page for more.
+
+Docker
+~~~~~~
+
+Run the following code snippet to generate a `Dockerfile <https://docs.docker.com/engine/reference/builder/>`_.
+This is a file that defines how to build a Docker image.
+
+(This requires having `Docker <https://docs.docker.com/get-docker/>`_ installed)
+
+.. code-block:: bash
+
+    neurodocker generate docker --pkg-manager apt \
+        --base-image neurodebian:buster \
+        --ants version=2.3.4 \
+        --miniconda version=latest conda_install="nipype notebook" \
+        --user nonroot
+
+You should see a block of text appear in the console. This is the Dockerfile generated
+by *Neurodocker*. Instructions in the Dockerfile are ordered in the same way as the
+arguments in the command-line. To build a Docker image with this, save the output to a
+file in an empty directory, and build with :code:`docker build`:
+
+.. code-block:: bash
+
+    mkdir docker-example
+    cd docker-example
+    neurodocker generate docker --pkg-manager apt \
+        --base-image neurodebian:buster \
+        --ants version=2.3.4 \
+        --miniconda version=latest conda_install="nipype notebook" \
+        --user nonroot > Dockerfile
+    docker build --tag nipype-ants .
+
+Then, you can start a Jupyter Notebook with the following command. This will mount
+the current working directory to :code:`work` within the container, so any files you
+create in this directory are saved. If we had not mounted this directory, all of the files
+created in :code:`/work` would be gone after the container was stopped.
+
+.. code-block:: bash
+
+    docker run --rm -it --workdir /work --volume $PWD:/work --publish 8888:8888 \
+        nipype-ants jupyter-notebook --ip 0.0.0.0 --port 8888
+
+Feel free to create a new notebook and :code:`import nipype`.
+
+Singularity
+~~~~~~~~~~~
+
+The only difference between this command and the one above is :code:`neurodocker generate singularity`
+versus :code:`neurodocker generate docker`. This will be the case when generating the
+majority of containers. The code block below generates a
+`Singularity definition file <https://sylabs.io/guides/3.7/user-guide/definition_files.html>`_.
+This file can be used to create a Singularity container.
```

Comment:
```suggestion
In most cases the only difference between generating Dockerfile and 
`Singularity definition file <https://sylabs.io/guides/3.7/user-guide/definition_files.html>`_ (the file that is used to create a Singularity container) is in  
a form of :code:`neurodocker generate` command,  `neurodocker generate singularity` has to be used instead of :code:`neurodocker generate docker`.
```